src/components/calculator.js

```diff
@@ -1,39 +1,60 @@
 import React from 'react';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: 0,
+      next: '',
+      operation: '',
+    };
+    this.handleClicks = this.handleClicks.bind(this);
```

Comment:
- [ ] Please The Calculator component (components/Calculator.js) should be refactored from a class-based component to a functional component using [hooks](https://reactjs.org/docs/hooks-intro.html)
![image](https://user-images.githubusercontent.com/74744787/182923158-c89aef89-0db0-4386-9d37-b66ab4d0270c.png)
- should look like this
```javascript
// ./src/components/Calculator.js

import React from 'react';

const Calculator = () => (
    // reference as a js object
      <div>
       // Calculator app design goes here
      </div>
);

export default Calculator; // export Calculator component
```

Reply:
@roshan-bajgain it was a mistake, I only wanted to request a review  for logic/handle event.

Reply:
@Roseokpe, I highly recommended you please create a new pull request for every review.